src/client/pages/StaffViewConfirmationPage/index.js

```diff
@@ -72,9 +74,9 @@ function StaffViewConfirmationPage(props) {
             <span className="badge completed">
               {t("staff-view-confirmation.completed.status")}
             </span>
-            {/* TODO(kalvin): make last name real */}
             <p>
-              {t("staff-view-confirmation.completed.last-name")}
+              {t("staff-view-confirmation.completed.last-name")} +
+              userData.lastName
```

Comment:
It looks like this isn't showing up correctly for some reason.

Reply:
whoops, i’ve been using renderNonTransContent for retrocert pages with Trans components, so those snapshots aren’t actually rendering inside some components :grimacing: fixing…